extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -613,6 +617,11 @@ public IngestTester testIngestQuery()
     return new IngestTester();
   }
 
+  public static CounterSnapshotBuilder channelCountersWith()
```

Comment:
Nit: syle I think CounterSnapshotBuilder can be a new class of its own and then this method gets pushed there.

Reply:
Sure, I will move it out.